app/controllers/projects_controller.rb

```diff
@@ -152,4 +156,9 @@ def ensure_owner_or_admin
         redirect_to projects_path
       end
     end
+
+    def get_order_param
+      return 'created_at desc' if params[:sort_by] == 'latest'
+      return 'volunteers.count asc' if params[:sort_by] == 'volunteers_needed'
```

Comment:
I would assume "volunteers_needed" would be `desc` so you could see the projects with the most demand. WDYT?

Reply:
I was thinking that "volunteers needed" == "project doesn't have enough volunteers" so, show the ones with the least number of volunteers. Do you think that's misleading somehow?

Reply:
Ahh, nevermind, I'm wrong! I was thinking of `number_of_volunteers` and missed that this was `volunteers.count`